Gems/ROS2/Code/Source/Lidar/LidarSensorConfiguration.cpp

```diff
@@ -87,9 +94,33 @@ namespace ROS2
         }
     }
 
+    AZStd::vector<AZStd::string> LidarSensorConfiguration::GetAvailableModels()
+    {
+        AZStd::vector<AZStd::string> result;
+        for (auto model : m_availableModels)
+        {
+            auto temp = LidarTemplateUtils::GetTemplate(model);
+            result.push_back({temp.m_name});
+        }
+        return result;
```

Comment:
The performance here is not of any concern whatsoever so I don't think so, that'd lead to redundancy.
Maybe we can keep their model names instead of models but that seems inelegant.